test/decl/func/operator.swift

```diff
@@ -147,7 +147,7 @@ postfix prefix func ++(x: Int) {} // expected-error {{attribute 'postfix' cannot
 
 // Don't allow one to define a postfix '!'; it's built into the
 // language.
-postfix operator!  // expected-error {{cannot declare a custom postfix '!' operator}}
+postfix operator!  // expected-error {{cannot declare a custom postfix '!' operator}} expected-error {{expected operator name in operator declaration}}
```

Comment:
@jtbandes Perhaps? I didn't realize until seeing that line that the lack of whitespace is legal, even, but I thought I ought to leave the existing test alone if it wasn't part of my change.